reana_commons/specification.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of REANA.
-# Copyright (C) 2022 CERN.
+# Copyright (C) 2022, 2023 CERN.
```

Comment:
Can you please also add 2023 to the global `LICENSE` file as well as update the `copyright = ...`  line in the `docs/conf.py` file?